README.md

```diff
@@ -0,0 +1,17 @@
+# Terraform Provider for Batch.sh
+
+* Website: [batch.sh](https://batch.sh)
+* Support Slack
```

Comment:
Yank - add email `support@batch.sh` or point users to open a GH issue.

Reply:
Also point to terraform docs on docs.batch.sh.

Reply:
Done!